### PR TITLE
Lar/master

### PR DESCRIPTION
**Pull request by lindseyreno**
This is my lunch level calculator.

### CODE REVIEW

File: calculator_bronze.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+puts "Hello! Welcome to my calculator!\nWhat kind of math operation would you like to use?"
+
+#what operation do they want to use? Get user input
+
+operation = gets.chomp
+
+#make an if/elsif statement for each operation. Add, subtract, multiply, divide
+if (operation.downcase == "addition") || (operation.downcase == "add") || (operation == "+")
```
Comment (ghost) [inline calculator_bronze.rb:8]: This is a good use of if/elsif/else statements along with those ||'s and =='s. Maybe consider using case (switch) statements (http://www.codecademy.com/glossary/ruby/switch-statement), but these if/elsif/else statements work just fine.

File: calculator_bronze.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
+puts "Hello! Welcome to my calculator!\nWhat kind of math operation would you like to use?"
```
Comment (ghost) [inline calculator_bronze.rb:1]: Cool! Nice use of the line break \n. Consider adding to the puts statement so the user knows what options to choose from and know what to type in (+, -, *, etc).

File: calculator.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+# Do the math and print the equation and the result
+
+if operation == "add"
+  puts "#{first_number} + #{second_number} = #{first_number + second_number}"
+elsif operation == "subtract"
+  puts "#{first_number} - #{second_number} = #{first_number - second_number}"
+elsif operation == "multiply"
+  puts "#{first_number} * #{second_number} = #{first_number * second_number}"
+elsif operation == "divide"
+  puts "#{first_number} / #{second_number} = #{first_number / second_number} Remainder #{first_number % second_number}"
```
Comment (ghost) [inline calculator.rb:39]: So when the number doesn't neatly divide into other number, you tell the user the remainder. Yyyyeahh!

Maybe think about how you would be able to tell the user the answer without a remainder (for example,  "4 / 3 = 1.333"). 

(hint: look at floats in Ruby and in particular the to_f method http://ruby.bastardsbook.com/chapters/numbers/)

**Comment by ghost** [comment]
Hi there! I'm from cohort[1] :cd: 

Good job on making my job easy with nice and readable code, and congratulations on getting to lunch! :pizza: 

Maybe think about ways you can make your program into a loop so that once a user gets their numbers crunched, the user can crunch some more numbers. Maybe with a while loop? http://www.codecademy.com/glossary/ruby/loops

BTW, the Ruby Docs are amazing (http://ruby-doc.org/core-2.2.2/) and got me through most of Ada, but I'm purposely included other links here since I found the docs hard to read in the beginning. But...the docs are useful for sure.

**Comment by lindseyreno** [comment]
Sorry its taken me so long to respond. I really appreciate your comments, they're very constructive and I liked the links that aren't the Ruby Docs.  Sometimes they can be a bit confusing.  I'm definitely going to try the switch statements.

**Pull request merged**